sos/plugins/samba.py

```diff
@@ -33,6 +33,10 @@ def setup(self):
             "/var/log/samba/log.winbindd-dc-connet",
             "/var/log/samba/log.wb-*"
         ])
+
+        if self.get_option("all_logs"):
+            self.add_copy_spec("/var/log/samba/")
+
```

Comment:
from looking at the implementation of add_copy_spec_size, it does not take an array of copyspecs, (in contrast to add_copy_spec). hence I need separate calls.